nylas/client/client.py

```diff
@@ -186,7 +195,11 @@ def token_for_code(self, code):
             self.access_token_url, data=urlencode(args), headers=headers
         ).json()
 
-        self.access_token = resp[u"access_token"]
+        self.access_token = resp["access_token"]
+        return resp
+
+    def token_for_code(self, code):
+        self.send_authorization(code)
```

Comment:
Does anything need to happen with the `resp` returned from `send_authorization`?

Reply:
@maxwell-schwartz no, this was just to fulfill #189 where the user would like to have the full payload made available to them rather than us stripping the access token from the response and returning the access token.